src/current/_data/releases.yml

```
@@ -5352,36 +5352,3 @@
     docker_arm_limited_access: true
   source: true
   previous_release: v23.2.0-rc.1
```

```suggestion
  previous_release: v23.2.0-rc.1
  
  - release_name: v23.1.14
  major_version: v23.1
  release_date: '2024-01-17'
  release_type: Production
  go_version: go1.21
  sha: b95f2225b09cf25f9f2be31ef6a6f65c73a1b081
  has_sql_only: true
  has_sha256sum: true
  mac:
    mac_arm: true
    mac_arm_experimental: true
    mac_arm_limited_access: false
  windows: true
  linux:
    linux_arm: true
    linux_arm_experimental: false
    linux_arm_limited_access: false
    linux_intel_fips: true
    linux_arm_fips: false
  docker:
    docker_image: cockroachdb/cockroach
    docker_arm: true
    docker_arm_experimental: false
    docker_arm_limited_access: false
  cloud_only: true
  cloud_only_message_short: 'Available only for CockroachDB Cloud clusters'
  cloud_only_message: >
      This version is currently available only for CockroachDB Cloud clusters.
      To request to upgrade a CockroachDB self-hosted cluster to this version, [contact support](https://support.cockroachlabs.com/hc/en-us/requests/new).
  source: true
  previous_release: v23.1.13
```
Sorry for that bad conflict resolution, this fixes it